config/localization/en/app.json

```diff
@@ -800,12 +801,15 @@
       "REFER_FOR_DISCOUNTS_FIRST_ORDER": "Receive up to {AMOUNT_USD} in fee discounts after placing your first order!",
       "REFER_FOR_DISCOUNTS": "Refer a friend and they can receive up to ${AMOUNT_USD} in discounts.",
       "REFEREE_BENEFITS": "The person you refer will save up to ${AMOUNT_USD} in fees.",
+      "REMAINING": "remaining",
```

Comment:
should probably be {X} remaining in case other languages put the number behind the word, no?

Reply:
agreeeed

Reply:
TODO when im in the mood for it later